### PR TITLE
activate prow test

### PR DESCRIPTION
**Pull request by dongwangdw**
Signed-off-by: dongdwdw <dongdwdw@cn.ibm.com>

### CODE REVIEW

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/8#" title="Author self-approved">dongwangdw</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **vincent-pli** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @vincent-pli` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: app.py
Comment (vincent-pli) [inline app.py:11]: Great, I like Tiger

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/8#" title="Author self-approved">dongwangdw</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/8#issuecomment-914861041" title="Approved">xq2005</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **vincent-pli** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by xq2005 — APPROVED** [review]

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/8#" title="Author self-approved">dongwangdw</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/8#pullrequestreview-748580456" title="Approved">xq2005</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **vincent-pli** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**
